service-api/src/main/java/greencity/dto/location/EditLocationDto.java

```diff
@@ -2,15 +2,24 @@
 
 import greencity.constant.ErrorMessage;
 import greencity.constant.ValidationConstant;
-import lombok.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 
-import javax.validation.constraints.*;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
 
 @Getter
 @Setter
 @AllArgsConstructor
 @NoArgsConstructor
 @EqualsAndHashCode
+@Builder
```

Comment:
Why you added builder here?? If you already have the setters ?

Reply:
edited